### PR TITLE
Add language settings and global processor settings closes #76

### LINKED ISSUE #76: adding languages list configuration
looking at the execution of ocrmypdf I could see is called as:

`ocrmypdf --redo-ocr -q - - `

that calls tesseract with the "eng" language:

`tesseract -l eng -c textonly_pdf=1 /tmp/com.github.ocrmypdf.cuwitjup/000002_ocr.png /tmp/com.github.ocrmypdf.cuwitjup/000002_ocr_tess pdf txt`

It would be useful to specify in some way a list of languages to pass to tesseract.

The best option would be a global/user settings page that let you specify the languages.
Anyway, since I use Docker and many setting are passed trough ENV variables, a much simpler alternative would be to specify lang list in and ENV variable, such as:

`WORKFLOW_OCR_LANGS=eng+ita+deu`

that should become:

`ocrmypdf -l eng+ita+deu --redo-ocr -q - -`

### PR DESCRIPTION
**Pull request by R0Wi**
What's still missing:

- [ ]  Localization
- [x]  Fix (unit-)tests
- [x]  Global admin page not yet functional
- [x]  Additional integration tests for VUE components
- [x]  Remove NC22 version and mark as backport
- [x] Increase Coverage
- [x] Add docs
- [x] Fix code smells reported by SonarQube

### CODE REVIEW

**Comment by R0Wi** [comment]
@bahnwaerter ready for review so far. Would be happy if you could give me some feedback :rocket: 

Btw: Sonarqube seems to scan an old version of this branch and i don't know why... so let's ignore the codesmells so far

File: Makefile [hunk 1]
```diff
@@ -87,10 +84,19 @@ composer-build:
 	composer install --no-dev --prefer-dist
 
 # Installs npm dependencies
-.PHONY: npm
-npm:
+.PHONY: npm-install
+npm-install:
+ifeq (,$(wildcard $(CURDIR)/package.json))
+	cd js && $(npm) npm run dev-install
```
Comment (bahnwaerter) [inline Makefile:90]: Do not append `npm` as parameter.
```suggestion
	cd js && $(npm) run dev-install
```

**Review by bahnwaerter — CHANGES_REQUESTED** [review]
I've reviewed and tested this patch. The tests have been done on a fresh Nextcloud 23.0.1 instance, where everything worked as expected.

Please take a look at the three marked code parts and resolve the annotated comments and suggestions. After that, this patch is ready for merging.

File: Makefile (hunk 1 above)
Comment (R0Wi) [reply Makefile:90]: 🙈

**Comment by R0Wi** [comment]
@bahnwaerter thx for feedback, did some adjustments in 834dd26a7e56641e590c12b76138e32e3c61bb09, please let me know if that works now 🚀

**Review by bahnwaerter — APPROVED** [review]
Now, everything seems fine. Ready for merging ...

**Comment by R0Wi** [comment]
The backport to `stable22` failed:
```
The process '/usr/bin/git' failed with exit code 1
```
To backport manually, run these commands in your terminal:
```bash
# Fetch latest updates from GitHub
git fetch
# Create a new working tree
git worktree add .worktrees/backport-stable22 stable22
# Navigate to the new working tree
cd .worktrees/backport-stable22
# Create a new branch
git switch --create backport-85-to-stable22
# Cherry-pick the merged commit of this pull request and resolve the conflicts
git cherry-pick ---mainline 1 7d9e98f81a2d24b66aec0d2521eb3301fcc43ce1
# Push it to GitHub
git push --set-upstream origin backport-85-to-stable22
# Go back to the original working tree
cd ../..
# Delete the working tree
git worktree remove .worktrees/backport-stable22
```
Then, create a pull request where the `base` branch is `stable22` and the `compare`/`head` branch is `backport-85-to-stable22`.

**Comment by R0Wi** [comment]
The backport to `stable23` failed:
```
The process '/usr/bin/git' failed with exit code 1
```
To backport manually, run these commands in your terminal:
```bash
# Fetch latest updates from GitHub
git fetch
# Create a new working tree
git worktree add .worktrees/backport-stable23 stable23
# Navigate to the new working tree
cd .worktrees/backport-stable23
# Create a new branch
git switch --create backport-85-to-stable23
# Cherry-pick the merged commit of this pull request and resolve the conflicts
git cherry-pick ---mainline 1 7d9e98f81a2d24b66aec0d2521eb3301fcc43ce1
# Push it to GitHub
git push --set-upstream origin backport-85-to-stable23
# Go back to the original working tree
cd ../..
# Delete the working tree
git worktree remove .worktrees/backport-stable23
```
Then, create a pull request where the `base` branch is `stable23` and the `compare`/`head` branch is `backport-85-to-stable23`.

**Pull request merged**